bookings/staff.go

```
@@ -0,0 +1,47 @@
+package bookings
+
+import (
+	"fmt"
+	zoho "github.com/schmorrison/Zoho"
+)
+
+func (c *API) FetchStaff(serviceID zoho.Parameter, staffID zoho.Parameter) (data StaffResponse, err error) {
+	endpoint := zoho.Endpoint{
+		Name:         FetchStaffModule,
+		URL:          fmt.Sprintf("https://www.zohoapis.%s/bookings/v1/json/%s", c.ZohoTLD,FetchStaffModule),
+		Method:       zoho.HTTPGet,
+		ResponseData: &StaffResponse{},
+		URLParameters: map[string]zoho.Parameter{
+			"filter_by": "",
+		},
+	}
+	if serviceID != "" {
+		endpoint.URLParameters["service_id"] = serviceID
+	}
+	if staffID != "" {
+		endpoint.URLParameters["staff_id"] = staffID
+	}
```

Do you need to return an error here if those parameters are empty?

---

I see it returns a list. NVM